akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -361,6 +363,63 @@ binding, but it should still be possible to deserialize old data with Jackson.
 
 It's a list of class names or prefixes of class names.
 
+## Rolling updates
```

Comment:
```suggestion
## Rolling upgrade
```

Since we are talking about updating a version, I think **upgrade** is a better term. Like in upgrade/downgrade.

Reply:
I'm sure both are used rather interchangeably, but I prefer update since it's not always better (could be introducing new bugs).

External data point https://kubernetes.io/docs/tutorials/kubernetes-basics/update/update-intro/